enterprise/internal/authz/bitbucketserver/authz.go

```diff
@@ -43,6 +46,12 @@ func newAuthzProvider(
 		return nil, nil
 	}
 
+	logger := log.Scoped("newAuthzProvider", "")
+	if errLicense := licensing.Check(licensing.FeatureACLs); errLicense != nil {
+		logger.Error("Check license for ACLs(BitBucket)", log.Error(errLicense))
+		return nil, errLicense
```

Comment:
We only need to either return or log the error, but not both unless we have a very good reason.
```suggestion
		return nil, errLicense
```